### PR TITLE
Added a call API

### PR DESCRIPTION
**Pull request by MrSp4rX**

### CODE REVIEW

File: bomber.py [hunk 1]
```diff
@@ -682,6 +682,12 @@ def getapi(pn,lim,cc):
...
+        ''').read()
+        return rd.find("1") != -1
```
Comment (TheSpeedX) [inline bomber.py:689]: Code improperly placed

**Comment by TheSpeedX** [comment]
Redirect Your PR to dev branch and do changes as per dev branch

**Review by 0n1cOn3 — APPROVED** [review]
#Dev

**Comment by TheSpeedX** [comment]
PR closed due to inactivity.

**Pull request closed without merging**